Lib/multiprocessing/connection.py

```diff
@@ -41,6 +42,7 @@
 BUFSIZE = 8192
 # A very generous timeout when it comes to local connections...
 CONNECTION_TIMEOUT = 20.
+WSA_OPERATION_ABORTED = 995
```

Comment:
It is the same as `_winapi.ERROR_OPERATION_ABORTED`.

Reply:
Now I'm confused. I don't recall which doc I was looking to. WriteFile() is documented to return ``ERROR_OPERATION_ABORTED`` when it's aborted: https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-writefile